packs/ocm-cluster-manager-v0.13.2/charts/ocm-cluster-manager/templates/cluster_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-manager
+subjects:
+- kind: ServiceAccount
+  name: cluster-manager
+  namespace: open-cluster-management
```

Comment:
The pack allows user to change the namespace through pack.namespace. But, the namespaces are hard coded in some of the templates. As a standard, the helm chart templates should use the release namespace to allow user customizations.